Invoking command in '/home/owenb/workareas/CAVREL/gps-node/src/py_gpsnode': PYTHONPATH=/home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/prefix_override:/home/owenb/workareas/CAVREL/gps-node/install/py_gpsnode/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/py_gpsnode build --build-base /home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/build install --record /home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/install.log --install-scripts $base/lib/py_gpsnode --single-version-externally-managed
Invoked command in '/home/owenb/workareas/CAVREL/gps-node/src/py_gpsnode' returned '0': PYTHONPATH=/home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/prefix_override:/home/owenb/workareas/CAVREL/gps-node/install/py_gpsnode/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../../build/py_gpsnode build --build-base /home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/build install --record /home/owenb/workareas/CAVREL/gps-node/build/py_gpsnode/install.log --install-scripts $base/lib/py_gpsnode --single-version-externally-managed
